packages/react-composites/src/composites/common/CaptionsBanner.tsx

```diff
@@ -1,39 +1,69 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 /* @conditional-compile-remove(close-captions) */
-import React from 'react';
+import React, { useState } from 'react';
 /* @conditional-compile-remove(close-captions) */
 import { _CaptionsBanner } from '@internal/react-components';
 /* @conditional-compile-remove(close-captions) */
 import { _DrawerMenu, _DrawerMenuItemProps, _DrawerSurface } from '@internal/react-components';
 /* @conditional-compile-remove(close-captions) */
-import { Stack } from '@fluentui/react';
+import { mergeStyles, Stack } from '@fluentui/react';
+/* @conditional-compile-remove(close-captions) */
+import { CaptionsSettingModal } from './CaptionsSettingModal';
+/* @conditional-compile-remove(close-captions) */
+import { CaptionsBannerMoreButton } from './CaptionsBannerMoreButton';
 /* @conditional-compile-remove(close-captions) */
 import { useAdaptedSelector } from '../CallComposite/hooks/useAdaptedSelector';
 /* @conditional-compile-remove(close-captions) */
 import { useHandlers } from '../CallComposite/hooks/useHandlers';
 /* @conditional-compile-remove(close-captions) */
 import { _captionsBannerSelector } from '@internal/calling-component-bindings';
+
 /* @conditional-compile-remove(close-captions) */
 /** @private */
 export const CaptionsBanner = (): JSX.Element => {
-  /* @conditional-compile-remove(close-captions) */
   const captionsBannerProps = useAdaptedSelector(_captionsBannerSelector);
-  /* @conditional-compile-remove(close-captions) */
```

Comment:
Let's still CC the inner part and return undefined if that is not CC, the idea is not to have export {} at the end